README.md

```diff
@@ -163,7 +190,7 @@ Restart server after making any changes to settings
 
 ### CAS Login Settings
 
-`CAS_LOGIN_ENABLED`: Enable login via CAS server (default: True)
+`CAS_LOGIN_ENABLED`: Enable login via CAS server (default: False)
```

Comment:
Could you add this change to the change log as a breaking change? As this might affect end user who are using default value.

Reply:
Added to change log along with the 3 renamed env variables
Also switched `SAML_LOGIN_ENABLED` to disabled by default. (Will still be on used by docker-compose by default)